compiler/modules/front/codeGen.bal

```diff
@@ -579,25 +579,22 @@ function codeGenAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environ
 }
 
 function codeGenCompoundAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environment env, CompoundAssignStmt stmt) returns CodeGenError|StmtEffect {
-    var { lValue, rexpr , op, pos} = stmt;
+    var { lValue, rexpr , op, pos } = stmt;
     if lValue is VarRefExpr {
         return codeGenCompoundAssignToVar(cx, startBlock, env, lValue, rexpr, op, pos);
     }
     else {
-        return codeGenCompoundAssignToMember(cx, startBlock, env, lValue, rexpr, op, pos);
+        return err:unimplemented("compound assignment to memebers not yet implemented");
```

Comment:
Get rid of this by having the right definition of CompoundAssignStmt

Reply:
fixed